### PR TITLE
Added the lazy camera look ahead script

### PR DESCRIPTION
**Pull request by xxfast**
this is a replacement camera for lazy camera

### CODE REVIEW

**Comment by xxfast** [comment]
[Game Mechanic; Camera Look ahead](https://trello.com/c/Fl19UNuj/45-game-mechanic-camera-look-ahead)

File: Assets/Scripts/LazyCameraLookAhead.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	public float lazinessFactor = 0.5f;
+	public GameObject target;
+
+	void Update () {
+		// Camera follows only if the target is assigned
+		if (target!=null) {
+			Vector3 velocity = Vector3.zero;
+			Vector3 forward = target.transform.forward * 10.0f;
+			Vector3 direction = target.GetComponent<Rigidbody2D> ().velocity;
+			Vector3 needPos = target.transform.position - forward + (direction * howFarToLookAhead);
```
Comment (Xavier-Gibbon) [inline Assets/Scripts/LazyCameraLookAhead.cs]: There needs to be a limit to how far the camera moves. If there is no limit, then the camera can leave the player behind if they travel far enough

Comment (xxfast) [inline Assets/Scripts/LazyCameraLookAhead.cs]: Hmm yeah thats a good catch. I'll right a function to dampen the camera movement

**Comment by xxfast** [comment]
[https://media.giphy.com/media/llRrcEsWiifXG/giphy.gif](url)

**Pull request merged**